### PR TITLE
(0.22.0) Allow building of aarch64 for all current releases

### PR DESCRIPTION
**Pull request by pshipton**
Replay of https://github.com/eclipse/openj9/pull/10614 for the 0.22 release.

### CODE REVIEW

**Comment by keithc-ca** [comment]
The comments should be updated here too (see https://github.com/eclipse/openj9/pull/10614#discussion_r489574698).

**Comment by pshipton** [comment]
Comments updated.

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy [hunk 1]
```diff
@@ -40,10 +40,10 @@
  *              win_x86-64_cmprssptrs,
  *              win_x86 (Java 8 support only),
  *              zos_390-64_cmprssptrs (Java 11 support only),
- *              osx_x86-64 (Java 8 and Java 11 support only),
- *              osx_x86-64_cmprssptrs (Java 8 and Java 11 support only),
- *              aarch64_linux (Java 8 and Java 11 support only),
- *              aarch64_linux_xl (Java 8 and Java 11 support only)
+ *              osx_x86-64,
+ *              osx_x86-64_cmprssptrs,
```
Comment (keithc-ca) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy:44]: See https://github.com/eclipse/openj9/pull/10614#discussion_r489604215.

Comment (pshipton) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy]: See if you like the latest change, and then I'll change the other PR to match.

Comment (pshipton) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy]: Updated again.

**Review by keithc-ca — APPROVED** [review]

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy (hunk 1 above)
Comment (keithc-ca) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy]: That works for me.

Comment (pshipton) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy]: Made another change, pls review again.

Comment (pshipton) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy]: @AdamBrousseau I'm still not sure if this is correct, can you pls review.

Comment (pshipton) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy]: @keithc-ca Adam isn't available to review in the short term. Are you ok to merge as-is, and we can make any further corrections later? I'd like to run a jdk15 aarch64 build which is blocked by getting this merged, and this has already been delayed from yesterday.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**